Q5.js

```diff
@@ -0,0 +1,18 @@
+// in the webTechs array check if Sass exists in the array and if it exists print 'Sass is a CSS preprocess'. If it does not exist add Sass to the array and print the array.
```

Comment:
@Fakhara77 you are not doing what the questions asks you to do

Reply:
sorry,now i have solved.next time i willread carefully.